tests/test_ImageCollection.py

```diff
@@ -340,6 +357,16 @@ def test_reduce_interval_with_non_existing_reducer_and_properties(self, jaxa_rai
         with pytest.raises(AttributeError):
             ic.geetools.reduceInterval("toto")
 
+    def test_reduce_interval_with_empty_days(self, s2_sr):
+        ic = s2_sr.filterDate("2021-01-01", "2021-01-07")
+        # Assert that this
+        try:
```

Comment:
I think you don't need the pytest.fail here. simply call reduced in a feature_collection_regression check and you'll be fine. (it's a new feature of pytest_ee it can be used exactly like a data_regression fixture (example here: https://github.com/gee-community/pytest-gee/blob/4ca9b9077ae76ca1b35e3e98869b05d92ef6a591/tests/test_pytest_gee.py#L59)

alternatively we could also vastly reduce the size of the payload by simply requesting the size of the List right . In this test what we want to test is that the empty imagercollections will be dropped so we know how many ImageCollections should appear.

Reply:
In this case, you say to do something like this?

```python
def test_reduce_interval_with_empty_days(self, s2_sr):
    ic = s2_sr.filterDate("2021-01-01", "2021-01-07")
    expectedSize = ic.geetools.groupInterval(duration=1, unit="day").size().getInfo()
    resultSize = ic.geetools.reduceInterval("mean", duration=1, unit="day").size().getInfo()
    assert resultSize == expectedSize
```

Reply:
close, more something like; 
```python 
def test_reduce_interval_with_empty_days(self, s2_sr):
    ic = s2_sr.filterDate("2021-01-01", "2021-01-07")
    resultSize = ic.geetools.reduceInterval("mean", duration=1, unit="day").size().getInfo()
    assert resultSize == 6 # I don't know the exact number of days but that should be hard coded in the test